phpstan-baseline.neon

```diff
@@ -0,0 +1,11 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#1 \\$statisticTaskRow of method App\\\\Service\\\\GoalScheduler\\\\TaskProgressResult\\:\\:incrementValues\\(\\) expects array\\<string, mixed\\>, string given\\.$#"
+			count: 1
+			path: src/Service/GoalScheduler/TaskProgressResult.php
+
+		-
+			message: "#^Parameter \\#1 \\$statisticTaskRow of method App\\\\Service\\\\GoalScheduler\\\\TaskProgressResult\\:\\:initTableIfNotExists\\(\\) expects array\\<string, mixed\\>, string given\\.$#"
+			count: 1
+			path: src/Service/GoalScheduler/TaskProgressResult.php
```

Comment:
In the TaskProgressResult, in the line 39 (`$this->taskCalendarRepository->getStatsForPreviousTasks`), there is an issue with annotation of returning type. This method returns `array<int, array>` instead of `array<int, string>`. Try to fix it. I hope it will help :)